server/config/main.go

```diff
@@ -19,7 +20,10 @@ var (
 )
 
 type Configuration struct {
-	Secret string `json:"Secret"`
+	Secret                      string `json:"Secret"`
```

Comment:
The annotation is not required

```suggestion
	Secret                      string
```